Automattic/wp-calypso
# Remove <LinkInBioBanner> from /sites page

## Proposed Changes

Removes the Link in Bio banner we were showing on `/sites` when the user has a small number of sites and none of them are "link in bio" sites.

Context p1702904752618509-slack-C0347E545HR

For reference, these are what the banners looked like
![CleanShot 2023-12-18 at 05 05 32@2x](https://github.com/Automattic/wp-calypso/assets/1500769/c2230293-47a8-4398-818f-c352b044ed66)


![image (5)](https://github.com/Automattic/wp-calypso/assets/1500769/e5e0f0a6-0700-4132-bf4a-c14bab50d488)


## Testing Instructions



* Create a user with only one site
* Visit `/sites`
* Confirm link-in-bio banner is gone from grid and list mode
* Confirm that the `link-in-bio-banner-medium.jpg` and `link-in-bio-banner-small.jpg` images weren't used anywhere else.

## Pre-merge Checklist
